dashboards/Pingdom/dashboard.json

```diff
@@ -0,0 +1,505 @@
+														"scriptIntervalSecs": 56,
+														"runAsIdentity": "Default",
+														"_security": "signing",
+														"dataType": "object",
+														"columnOverrides": {
+															"value": "lastresponsetime",
+															"grouping": "name"
+														},
+														"profile": "Pingdom",
+														"script": "## Connection Info\n$uri = \"checks\"\n$results = Invoke-RestMethod -Uri ($url+$Uri) -Headers $headers -Method Get\n\n## Output results and sort by last response time\n$results.checks | sort lastresponsetime -Descending"
+													},
+													"signature": ""
+												}
+											},
+											"display": {
+												"range": "percentage",
+												"label": "auto",
+												"sort": null,
+												"customLabel": ""
+											}
+										},
+										"title": "Last Response Time",
+										"description": ""
+									}
+								]
+							}
+						],
+						"width": 50
+					}
+				]
+			},
+			{
+				"_type": "layout/column",
+				"columns": [
+					{
+						"width": 100,
+						"contents": [
+							{
+								"_type": "layout/list",
+								"contents": [
+									{
+										"_type": "tile/powershell-as-grid",
+										"config": {
+											"context": {},
+											"source": {
+												"scriptTimeoutSecs": 54,
+												"scriptIntervalSecs": 56,
+												"runAsIdentity": "Default",
+												"_security": "signing",
+												"dataType": "object",
+												"profile": "Pingdom",
+												"script": "## Connection Info\n$uri = \"checks\"\n$results = Invoke-RestMethod -Uri ($url+$Uri) -Headers $headers -Method Get\n\n## Output Results\n$results.checks",
+												"__signed": {
+													"payload": {
+														"scriptTimeoutSecs": 54,
+														"scriptIntervalSecs": 56,
+														"runAsIdentity": "Default",
+														"_security": "signing",
+														"dataType": "object",
+														"profile": "Pingdom",
+														"script": "## Connection Info\n$uri = \"checks\"\n$results = Invoke-RestMethod -Uri ($url+$Uri) -Headers $headers -Method Get\n\n## Output Results\n$results.checks"
+													},
+													"signature": ""
+												}
+											},
+											"display": {
+												"showHeaders": true,
+												"columnOverrides": {
+													"created": {
+														"template": "{{timeago(value)}}"
+													},
+													"lasterrortime": {
+														"template": "{{timeago(value)}}"
+													},
+													"lasttesttime": {
+														"template": "{{timeago(value)}}"
+													},
+													"lastresponsetime": {
+														"template": "{{timeago(value)}}"
+													},
+													"lastdownstart": {
+														"template": "{{timeago(value)}}"
+													},
+													"lastdownend": {
+														"template": "{{timeago(value)}}"
+													}
+												},
+												"customColumns": {},
+												"columns": [

```

Comment:
Maybe too many columns? Could just be me though.

Reply:
Left it unfiltered to showcase all that is returned.

Reply:
Date formatting isn't working too well?  everything says January 18th/19th 1970...

![image](https://user-images.githubusercontent.com/18680913/130096812-a815fd85-023f-4f08-b2a8-4e2fbb3e28e4.png)

And I agree, too many columns.

Reply:
Trimmed and fixed dates
![image](https://user-images.githubusercontent.com/45064152/130611922-60b14bd8-97ba-4f1b-bba9-73eb8fd4da91.png)